android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -401,6 +402,7 @@ public boolean onMenuOrKeyboardAction(int id, boolean fromMenu) {
         } else if (id == R.id.brave_speedreader_id) {
             enableSpeedreaderMode();
         } else if (id == R.id.brave_leo_id) {
+            BraveLeoUtils.verifySubscription(null);
```

Comment:
the `null` parameter seems to be the verify `success/failure` callback here.
How can we be sure this was not a failed verification?

Reply:
it's a callback to execute some action when we pulled subscription data from Google. No action needed in that case, everything is hooked up inside and interacts with native part for web-ui to know. The web-ui is going to assume that there is no subscription if it's failed.